.travis.yml

```diff
@@ -25,12 +25,24 @@ jobs:
         - go run build/ci.go install
         - go run build/ci.go test -coverage $TEST_PACKAGES
 
+    - stage: build
+      os: linux
+      dist: xenial
+      go: 1.13.x
```

Comment:
go 1.13

Reply:
I intentionally left the travis and appveyor ones as is, since we don't use those services and I figured no need to create gratuitous conflicts later on.